src/image.c

```diff
@@ -82,12 +87,53 @@ void image_thumbnail(struct image* image, size_t size, bool fill,
         offset_y = 0;
     }
 
+    // TODO: probably should be moved outside of this function
+    if (!cache_dir) {
+        cache_dir = getenv("XDG_CACHE_HOME");
+        if (cache_dir) {
+            cache_dir = strcat(cache_dir, "/swayimg");
+        } else {
+            cache_dir = getenv("HOME");
+            if (!cache_dir) {
+                goto thumb_create;
+            }
+            cache_dir = strcat(cache_dir, "/.swayimg");
```

Comment:
`HOME/.cache/swayimg`, but it would be better to have an option in the config file so the user could set it themselves.

Reply:
Adding this to config would be better for another PR i think, but I can try adding it here too...

Reply:
Yes, this can be taken out into a separate request.
But in any case, this PR should contain the configurable option for enabling/disabling the persistent cache.